rsconnect/main.py

```diff
@@ -2417,6 +2417,119 @@ def start_content_build(
         build_start(ce.remote_server, parallelism, aborted, error, all, poll_wait, debug)
 
 
+@cli.group(no_args_is_help=True, help="Interact with Posit Connect's system API.")
+def system():
+    pass
+
+
+@system.group(no_args_is_help=True, help="Interact with Posit Connect's system caches.")
+def caches():
+    pass
+
+
+# noinspection SpellCheckingInspection,DuplicatedCode
+@caches.command(
+    name="list",
+    short_help="List runtime caches present on a Posit Connect server.",
+)
+@click.option("--name", "-n", help="The nickname of the Posit Connect server.")
+@click.option(
+    "--server",
+    "-s",
+    envvar="CONNECT_SERVER",
+    help="The URL for the Posit Connect server.",
+)
+@click.option(
+    "--api-key",
+    "-k",
+    envvar="CONNECT_API_KEY",
+    help="The API key to use to authenticate with Posit Connect.",
+)
+@click.option(
+    "--insecure",
+    "-i",
+    envvar="CONNECT_INSECURE",
+    is_flag=True,
+    help="Disable TLS certification/host validation.",
+)
+@click.option(
+    "--cacert",
+    "-c",
+    envvar="CONNECT_CA_CERTIFICATE",
+    type=click.Path(exists=True, file_okay=True, dir_okay=False),
+    help="The path to trusted TLS CA certificates.",
+)
+@click.option("--verbose", "-v", is_flag=True, help="Enable verbose output")
```

Comment:
```suggestion
@server_args
```

Reply:
Oh, thanks! I figured there must be a more concise way to do that.